code/modules/mob/living/carbon/species.dm

```diff
@@ -245,9 +257,13 @@ var/global/list/playable_species = list("Human")
 		H.organs += OE
 		if(OE.grasp_id)
 			H.grasp_organs += OE
-
-	for(var/datum/organ/external/O in H.organs)
-		O.owner = H
+	for(var/organ as anything in H.cosmetic_organs_by_name)
```

Comment:
remove `as anything`

Reply:
@ShiftyRail its faster

Reply:
No it isn't. And it's not standard practice. Remove this or it won't be merged.